test/airbrake/utils_test.exs

```diff
@@ -48,4 +46,78 @@ defmodule Airbrake.UtilsTest do
              }
     end
   end
+
+  describe "detuple/1" do
+    test "converts tuples to lists" do
+      input = %Struct{baz: {1, 2}, qux: {:ok, "foobar"}}
+      expected = %Struct{baz: [1, 2], qux: [:ok, "foobar"]}
+
+      assert Utils.detuple(input) == expected
+    end
+
+    test "returns de-tupled data when input contains deeply nested tuples" do
+      input = %Struct{baz: [1, 2, {3, 4}], qux: %{foo: %{bar: {9, 9, 9, 9}}}}
+      expected = %Struct{baz: [1, 2, [3, 4]], qux: %{foo: %{bar: [9, 9, 9, 9]}}}
+
+      assert Utils.detuple(input) == expected
+    end
+
+    test "returns the original data when input is a map with no tuples" do
+      input = %{baz: 100, qux: 200}
+
+      assert Utils.detuple(input) == input
+    end
+
+    test "returns the original data when input is a list with no tuples" do
+      input = ["foo", "bar"]
+
+      assert Utils.detuple(input) == input
+    end
+
+    test "returns a list when input is a tuple" do
+      input = {:ok, "something"}
```

Comment:
Replace `"something"` with a tuple.  It'll look really weird, but I'd like to test the recursion here, too.